JS/app.js

```diff
@@ -0,0 +1,45 @@
+//Sec 1
+//1. Concatenation is to combine strings. Interpolation is using a variable to represent a value.
+console.log('hello '+'world.');
+const word = "hello";
+console.log(word + " world.");
+//2.Do Not Repeat Yourself. Program runs more efficient and easier to interpret by humans. Loops.
+
+//Sec 2
+ const a = 4;
+ const b = 53;
+ const c = 57;
+ const d = 16;
+ const e = 'Kevin';
+
+console.log(a < b);
+console.log(c > d);
+console.log('Name' === 'Name');
+console.log(a < b < c);
+console.log(a < a < d);
+console.log(e === 'Kevin');
+console.log(48 == '48');
+
+//Sec 3
+// Infinite Loop
+//Not Infinite Loop
+//Concatenat 20 "A" strings
+
+//Sec 4
+//While Loop the condition is evaluated before the loop is executed. Both loops are for repition and contain conditions.
```

Comment:
good -- a while loop depends on something, just any general thing, becoming false at some point.  that condition could be based on something external to the loop's code block.  a for loop contains all its control logic inside the parentheses.